doc/build/dts/bindings-intro.rst

```diff
@@ -0,0 +1,134 @@
+.. _dt-binding-compat:
```

Comment:
I don't understand what you mean by "top level page".

The reason why these pages don't show up in the left hand sidebar is because the DT pages are buried too deeply. That's why I added `..contents::` directives to several of them where I anticipate internal navigation being appreciated by readers.